imageroot/actions/create-module/20postgrestoken

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2025 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import agent
+import os
+import string
+import secrets
+
+try:
+    secrets_env = agent.read_envfile("secrets.env")
+except FileNotFoundError:
+    # Generate a new API token
+    alphabet = string.ascii_letters + string.digits
+    postgres_token = ''.join([secrets.choice(alphabet) for i in range(32)])
```

Comment:
This way  can be simpler to read and understand.
See: https://docs.python.org/3/library/secrets.html
```suggestion
    postgres_token = secrets.token_urlsafe(32)
```